src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateCoverageComponents.java

```diff
@@ -73,8 +73,13 @@ protected void runPipeline(final JavaSparkContext ctx) {
         final List<String> targetNames = analysisReadCounts.targets().stream().map(Target::getName).collect(Collectors.toList());
         final List<String> sampleNames = analysisReadCounts.columnNames();
         final double[][] counts = analysisReadCounts.counts().getData();
+        //TODO Mysterious bug when trying to use the Spark based SVD
```

Comment:
could you have this include the issue number in this comment?  That way someone who sees it in the future can look it up if this doesn't get fixed quickly.